doc/mkdocs/docs/api/basic_json/operator[].md

```diff
@@ -42,7 +48,8 @@ const_reference operator[](const json_pointer& ptr) const;
 
 1. reference to the element at index `idx`
 2. reference to the element at key `key`
-3. reference to the element pointed to by `ptr`
+3. see 2
```

Comment:
Nitpicky, but I'd rather repeat `Logarithmic in the size of the container.` here. The level of indirection for that short sentence confuses me.

Reply:
What about

2. & 3. reference to the element at key `key`

Reply:
He copy & pasted the message. I understand what he means.

Reply:
I understood it too, I'm suggesting

    2. & 3. reference to the element at key `key`

rather than

    2. reference to the element at key `key`
    3. reference to the element at key `key`

Reply:
I wish this would possible, but that does not work in Markdown lists.

If it's in the middle of a list, then the numbering breaks. If it's at the end of the list, then numbering is fine, but indentation looks odd.

Reply:
Ah, okay.  That's too bad.

Reply:
I have come across a "better lists" markdown extension. Maybe it is possible? I may look into that later if I remember.

Reply:
Except that the numbering is of course done by HTML `<ol>` tags. So probably not...